### PR TITLE
Cleanup mislabeled BSD license

### PR DESCRIPTION
**Pull request by tfoote**
This was propagated due to an invalid license in ament_lint which is proposed to be fixed at: https://github.com/ament/ament_lint/pull/209

### CODE REVIEW

File: ros2topic/ros2topic/verb/bw.py [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
-# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
-# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# modification, are permitted provided that the following conditions are met:
+
+#    * Redistributions of source code must retain the above copyright
+#      notice, this list of conditions and the following disclaimer.
+
```
Comment (jacobperron) [inline ros2topic/ros2topic/verb/bw.py]: Blank lines in the middle of the license look wrong

File: ros2topic/ros2topic/verb/delay.py [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
+#      notice, this list of conditions and the following disclaimer.
+
+#    * Redistributions in binary form must reproduce the above copyright
+#      notice, this list of conditions and the following disclaimer in the
+#      documentation and/or other materials provided with the distribution.
+
+#    * Neither the name of the copyright holder nor the names of its
+#      contributors may be used to endorse or promote products derived from
+#      this software without specific prior written permission.
+
```
Comment (jacobperron) [inline ros2topic/ros2topic/verb/delay.py]: More aesthetic fixes can be made in this license

File: ros2topic/ros2topic/verb/hz.py [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
+#      notice, this list of conditions and the following disclaimer.
+
+#    * Redistributions in binary form must reproduce the above copyright
+#      notice, this list of conditions and the following disclaimer in the
+#      documentation and/or other materials provided with the distribution.
+
+#    * Neither the name of the copyright holder nor the names of its
+#      contributors may be used to endorse or promote products derived from
+#      this software without specific prior written permission.
+
```
Comment (jacobperron) [inline ros2topic/ros2topic/verb/hz.py]: More aesthetic fixes can be made in this license

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15695/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/10375/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13360/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15931/)

**Pull request merged**